src/main/java/net/fabricmc/loom/configuration/providers/minecraft/MinecraftProvider.java

```diff
@@ -122,7 +122,7 @@ protected void initFiles() {
 	}
 
 	private void verifyJars() throws IOException, SignatureVerificationFailure {
-		if (GradleUtils.getBooleanProperty(getProject(), Constants.Properties.DISABLE_MINECRAFT_VERIFICATION)) {
+		if (GradleUtils.getBooleanProperty(getProject(), Constants.Properties.DISABLE_MINECRAFT_VERIFICATION, true)) {
 			LOGGER.info("Skipping Minecraft jar verification!");
```

Comment:
This is not fully related to the PR, but this block also needs to exit the method otherwise setting the property does nothing.

Reply:
Fixed directly in 1.11.